code/modules/mining/lavaland/necropolis_chests.dm

```diff
@@ -140,6 +139,49 @@
+			if(do_after(itemUser, 40, target = src))
+				itemUser.say("I swear to fulfill, to the best of my ability and judgment, this covenant:")
+			else
+				to_chat(itemUser, "<span class='warning'>The snake seems unsatisfied with your incomplete oath and returns to it's previous place on the rod, returning to its dormant, wooden state. You must stand still while completing your oath!</span>")
+				return
+			if(do_after(itemUser, 20, target = src))
+				itemUser.say("I will apply, for the benefit of the sick, all measures that are required, avoiding those twin traps of overtreatment and therapeutic nihilism.")
+			else
+				to_chat(itemUser, "<span class='warning'>The snake seems unsatisfied with your incomplete oath and returns to it's previous place on the rod, returning to its dormant, wooden state. You must stand still while completing your oath!</span>")
+				return
+			if(do_after(itemUser, 30, target = src))
+				itemUser.say("I will remember that I remain a member of society, with special obligations to all my fellow human beings, those sound of mind and body as well as the infirm.")
+			else
+				to_chat(itemUser, "<span class='warning'>The snake seems unsatisfied with your incomplete oath and returns to it's previous place on the rod, returning to its dormant, wooden state. You must stand still while completing your oath!</span>")
+				return
+			if(do_after(itemUser, 30, target = src))
+				itemUser.say("If I do not violate this oath, may I enjoy life and art, respected while I live and remembered with affection thereafter. May I always act so as to preserve the finest traditions of my calling and may I long experience the joy of healing those who seek my help.")
+			else
+				to_chat(itemUser, "<span class='warning'>The snake seems unsatisfied with your incomplete oath and returns to it's previous place on the rod, returning to its dormant, wooden state. You must stand still while completing your oath!</span>")
+				return
+			to_chat(itemUser, "<span class='notice'>The snake, satisfied with your oath, attaches itself and the rod to your forearm with an inseparable grip. Your thoughts seem to only revolve around the core idea of helping others, and harm is nothing more than a distant, wicked memory...</span>")
+			var/datum/status_effect/hippocraticOath/effect = itemUser.apply_status_effect(STATUS_EFFECT_HIPPOCRATIC_OATH)
+			effect.hand = itemUser.get_held_index_of_item(src)
+			src.flags_1 = NODROP_1 | DROPDEL_1
+			src.desc = "A short wooden rod with a mystical snake inseparably gripping itself and the rod to your forearm. It flows with a healing energy that dsiperses amongst yourself and those around you. "
+			src.icon_state = "talisman"
+			src.activated = TRUE
+		else
+			to_chat(user, "<span class='warning'>The snake carving seems to come alive, if only for a moment, before returning to it's dormant state, almost as if it finds you incabable of holding it's oath.</span>")
```

Comment:
>incabable

Reply:
i've driven every man in this city to where they need to go but you sir, you are just simply incabable.

Reply:
shoot let me fix